ruby/conclusion/project_ruby_final.md

```diff
@@ -16,12 +16,14 @@ This is a great project to have as a part of your portfolio going forward becaus
   3. Make it so you can save the board at any time (remember how to serialize?)
   2. Write tests for the important parts.  You don't need to TDD it (unless you want to), but be sure to use RSpec tests for anything that you find yourself typing into the command line repeatedly.
   3. Do your best to keep your classes modular and clean and your methods doing only one thing each.  This is the largest program that you've written, so you'll definitely start to see the benefits of good organization (and testing) when you start running into bugs.
-  4. Have fun!  Check out the [unicode characters](http://en.wikipedia.org/wiki/Chess_symbols_in_Unicode) for a little spice for your gameboard.
-  5. (Optional extension) Build a very simple AI computer player (perhaps who does a random legal move)
+  4. Unfamiliar with Chess? Check out some of the additional resources to help you get your bearings.
+  5. Have fun!  Check out the [unicode characters](http://en.wikipedia.org/wiki/Chess_symbols_in_Unicode) for a little spice for your gameboard.
+  6. (Optional extension) Build a very simple AI computer player (perhaps who does a random legal move)
 </div>
 
 ### Additional Resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 * [Illustrated rules of Chess](http://www.chessvariants.org/d.chess/chess.html)
+* [Chess notation](https://en.wikipedia.org/wiki/Chess_notation)
 * [A series of hints from RubyQuiz](http://rubyquiz.com/quiz35.html), where this was one of their challenges.  You don't need these.  Resist the temptation!
```

Comment:
**thought (non-blocking):** It might be time to reconsider our inclusion of the RubyQuiz hints or rewording the description. 

If this new PR specifically directs learners to check out the Additional Resources then we might want to be more clear that these RubyQuiz hints should be a last resort. We might want to strike them altogether. 

Non-blocking since this is really a separate issue but I will bring it up for discussion with the Ruby team.

Reply:
@dm-murphy @ChargrilledChook 

I had thought about this, but didn't want to make any additional changes in my initial commit.

Having reviewd this resource, it basically contains an answer. How would folks feel if we just flagged that in the link?

I would suggest the following:

```markdown
* [A discussion of this problem from RubyQuiz](http://rubyquiz.com/quiz35.html). This link contains a blueprint for how to complete this project. If you're feeling stuck or need a hint, you might want to visit the community discord for help instead of looking up the answer!```

I think it a) lets folks know that it does contain a blueprint, with the corollary that if you care about solving this problem, it's not a good resource to pursue and b) prompts folks that might be stuck to check out the discord for help instead.

Reply:
Nice, I really like the mention of visiting the community Discord for help! 👍 

One more thought on this one. I liked the original language of "You don't need these. Resist the temptation!" 

What about starting off the bullet point with something like "You won't need this. Resist the temptation! But here is [a discussion of this problem from RubyQuiz]..."

Feel free to play with all of this and commit what you think might be best. Thanks again for working through these iterations @crespire!